CMakeLists.txt

```diff
@@ -65,7 +80,29 @@ target_link_libraries(OpenDDW
 
 OPENDDS_TARGET_SOURCES(OpenDDW idl/std_qos.idl)
 
+INCLUDE(CMakePackageConfigHelpers)
+
+INSTALL(EXPORT ${PROJECT_NAME}
+	NAMESPACE ${PROJECT_TARGET_PREFIX}::
+	FILE ${PROJECT_TARGET_FILE}
+	DESTINATION ${CONFIG_FILE_INSTALL_LOCATION}
+)
```

Comment:
I think the issue is there are no targets in `OpenDDW` to export. Looking at my CMake work in OpenDDS as a reference, I'm calling [this in a helper function as I'm defining targets](https://github.com/iguessthislldo/OpenDDS/blob/410c7a45563904d4796b276ba215dadfdfaf8208/cmake/opendds_build_helpers.cmake#L63) which is adding the targets to `opendds_targets`. After all the targets are defined I'm calling `install(EXPORT opendds_targets ...)` [here](https://github.com/iguessthislldo/OpenDDS/blob/410c7a45563904d4796b276ba215dadfdfaf8208/CMakeLists.txt#L84-L87) in the root CMakeLists. So I think you need a [`install(TARGETS OpenDDW ...)`](https://cmake.org/cmake/help/latest/command/install.html#install-targets) before the `install(EXPORT)`.